### DISCUSSION TITLE
Problem when updating v4 to v5

### DISCUSSION THREAD

**Discussion by kkhanhluu** [mikro-orm/mikro-orm#2955] [Q&A] [answered]
I encountered this error while trying to update mikro orm v4 to v5
`
[ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './entity/BaseEntity' is not defined by "exports" in "@mikro-orm/core"
`
Here are my project's information: 
- Node: 16.*
- Typescript: 4.6.2 
- "@mikro-orm/cli": "5.1.1", "@mikro-orm/core": "5.1.1", "@mikro-orm/migrations": "5.1.1", "@mikro-orm/postgresql": "5.1.1", "@mikro-orm/reflection": "5.1.1"

Did someone face the similar error?

**Comment by tolgap** [comment, marked as answer]
Similar issue here: https://github.com/mikro-orm/mikro-orm/discussions/2915. They were able to solve it.

**Reply by kkhanhluu** [reply]
Thanks. I solved the problem :)
